tests/markdown_paragraph/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -35,6 +50,13 @@ post](http://jlongster.com/A-Prettier-Formatter))
 
 `;
 
+exports[`inline-nodes.md 2`] = `
+It removes all original styling[*](#styling-footnote) and ensures that all outputted code conforms to a consistent style. (See this [blog post](http://jlongster.com/A-Prettier-Formatter))
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It removes all original styling[\\*](#styling-footnote) and ensures that all outputted code conforms to a consistent style. (See this [blog post](http://jlongster.com/A-Prettier-Formatter))
```

Comment:
This \\* isn't a bug, is it?

Reply:
We escape all the `*` char to prevent mis-parsing from user's future typing, e.g.

```md
a * b = c
```
become
```html
<p>a * b = c</p>
```
but 

```md
a * b = c, e * f = g
```
become
```html
<p>a <em> b = c, e </em> f = g</p>
```